CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
 # Contributing guidelines
 
-For anyone who knows [Git](https://git-scm.com) and JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/CODE_OF_CONDUCT.md).
+For anyone who knows JavaScript basics: feel free to contribute to this repository! Our code is open source. Be sure to follow our [code of conduct](https://github.com/ScratchAddons/ScratchAddons/blob/master/CODE_OF_CONDUCT.md).
 
 ## Filing [issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues)
 
-Found a bug? Report it! Be sure to check if it is already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
+Found a bug? Report it in an issue! But first, be sure to check if it was already reported by using the search bar [here](https://github.com/ScratchAddons/ScratchAddons/issues). If there are no similar issues, you can create a new one. We will take a look at it.
 
 Also, feel free to ask questions, suggest new features, and more.
 
 ## Creating [pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
 
-Contributions are welcome! Be sure to file an issue first (see above). If you know a fix for that issue, or you know a fix for a different one, you can create a pull request. Fork this repository, create a new branch from the 'master' branch, and make your changes on the new branch. Now, create a pull request on the origin repository (ScratchAddons/ScratchAddons). We will review your pull request.
+Contributions are welcome! If you have something in mind, it's best if you file an issue first (see above). That way, we can discuss it before you start a pull request. If you know a fix for that issue (or any other issue), you can follow the steps below to create a pull request.
```

Comment:
> If you have something in mind, it's best if you file an issue first (see above). That way, we can discuss it before you start a pull request.

Could we make this a separate paragraph, and apply `**bold**` on it?

Reply:
Maybe I'm exaggerating, but I almost skip this paragraph while reading this, and I think it's one of the most important sentences of the whole file.
It could also be added as step 1, or anywhere else where it's more likely to be read.

Reply:
I'll make this step 1 of creating a pull request.